packages/amplify_api/analysis_options.yaml

```diff
@@ -1,6 +1,8 @@
 include: package:amplify_lints/flutter_lib.yaml
 
 analyzer:
+  exclude:
```

Comment:
Good question. It looks like that score comes from running analysis with our own file haha: https://github.com/dart-lang/pana/blob/master/lib/src/sdk_env.dart#L157. So it may actually help us!

Reply:
I feel like we just discovered a cheat code